etna/search/views.py

```diff
@@ -174,6 +174,30 @@ def paginate_api_result(
         return paginator, page, page_range
 
 
+data = {
+    "contentGroup1": "Search",  # The name of the content group - [Always has a value]
+    "customDimension1": "offsite",  # The reader type (options are "offsite",
+    "customDimension2": "",  # The user type and is private beta specific
+    "customDimension3": "",  # The page type - [Always has a value]
+    "customDimension4": "",  # Taxonomy topics for the page, delineated by semi-colons. Empty string if no value.
+    "customDimension5": "",  # This is the taxonomy sub topic where applicable. Empty string if not applicable.
+    "customDimension6": "",  # This is the taxonomy term where applicable. Empty string if not applicable.
+    "customDimension7": "",  # This is the time period where applicable. Empty string if not applicable.
+    "customDimension8": "",  # This is the sub time period where applicable. Empty string if not applicable.
+    "customDimension9": "",  # This is the entity type where applicable. Empty string if not applicable.
+    "customDimension10": "",  # This is the entity label where applicable. Empty string if not applicable.
+    "customDimension11": "",  # This is the catalogue repository where applicable. Empty string if not applicable.
+    "customDimension12": "",  # This is the catalogue level where applicable. Empty string if not applicable.
+    "customDimension13": "",  # This is the catalogue series where applicable. Empty string if not applicable.
+    "customDimension14": "",  # This is the catalogue reference where applicable. Empty string if not applicable.
+    "customDimension15": "",  # This is the catalogueDataSource where applicable. Empty string if not applicable.
+    "customDimension16": "",  # This is the availability condition category where applicable. Empty string if not applicable.
+    "customDimension17": "",  # This is the availability condition where applicable. Empty string if not applicable.
+    "customMetric1": "",  # This is the number of search results
+    "customMetric2": "",  # This is the number of search filters applied
+}
+
+
 class SearchLandingView(BucketsMixin, TemplateView):
     """
```

Comment:
We can reuse [DataLayerMixin](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/analytics/mixins.py#L6)  and add to this class.

For this implement method `get_gtm_content_group` to `return Search` - the content group